core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1364,6 +1364,47 @@ private static String toSql(RelNode root, SqlDialect dialect,
     sql(query).ok(expected);
   }
 
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6909">[CALCITE-6909]
+   * ClickHouse Dailect can not support accurate precision</a>.
+   */
+  @Test void testClickHouseCastDecimalPrecision() {
```

Comment:
The test cases here are ineffective for the changes.

Reply:
Thanks @NobiGo , I had changed the test case，before the pr would error out：
Expected: is "SELECT CAST(`product_id` AS DECIMAL(75, 12))\nFROM `foodmart`.`product`"
but: was "SELECT CAST(`product_id` AS DECIMAL(19, 12))\nFROM `foodmart`.`product`"